pyaedt/misc/Run_PyAEDT_Script.py_build

```diff
@@ -1,43 +1,47 @@
 """
 * * * This script is meant to run in IronPython within the Ansys Electronics Desktop * * *
-The script allows choosing a .py script to execute
+The script allows choosing a .py script to execute.
 
-It looks for a reference to a python interpreter in python_interpreter.bat
+It looks for a reference to a Python interpreter in python_interpreter.bat
 
-It will then use this python interpreter to execute the script.
+It will then use this Python interpreter to execute the script.
 See the declaration of the command variable to see the order in which arguments are passed to the script.
 
 The commands allow the launched script to still reference the project and design that was active when the script
 was launched as well as the AEDT instance that has them open.
 
 Author: Carlos A. Mulero Hernandez
 """
+import os
+import subprocess
+
 import ScriptEnv
-import clr
+
+# import clr
 # clr.AddReference('System.Windows.Forms')
 from System.Windows.Forms import OpenFileDialog
 
-import os
-import subprocess
-
 oProject = oDesktop.GetActiveProject()
 oDesign = oProject.GetActiveDesign()
-#choose file to launch
 
+# Choose file to launch
```

Comment:
```suggestion
# choose file to launch
```
It doesn't matter whether all comments in a file start with a lowercase letter or uppercase letter. They should just all be consistent.

Reply:
@PipKat  in this file this is the only comment line, so Ill stick with what it is for now.

Reply:
@kbhagat-ansys That's fine. I prefer starting them with an uppercased letter myself!